packages/react/src/patterns/blocks/ContentGroupCards/README.md

```diff
@@ -0,0 +1,114 @@
+import 'yourapplication.scss';
+
+const heading = 'Lorem ipsum dolor sit amet.';
+
+const items = [
+  {
+    heading: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
+    copy:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+    cta: {
+      href: 'https://www.example.com',
+    },
+  },
+  {
+    heading: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
+    copy:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+    cta: {
+      href: 'https://www.example.com',
+    },
+  },
+  {
+    heading: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
+    copy:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+    cta: {
+      href: 'https://www.example.com',
+    },
+  },
+  {
+    heading: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
+    copy:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+    cta: {
+      href: 'https://www.example.com',
+    },
+  },
+];
+
+function App() {
+  return <ContentGroupCards heading={heading} content={content} />;
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the ContentGroupCards styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+## Props
+
+| Name      | Required | Data Type | Default Value | Description                                          |
+| --------- | -------- | --------- | ------------- | ---------------------------------------------------- |
+| `heading` | YES      | String    | n/a           | Main heading of the pattern.                         |
+| `items`   | NO       | Array     | null          | Array of content group objects. See `content` below. |
+
+### content
+
+| Name      | Data Type | Description                                                |
+| --------- | --------- | ---------------------------------------------------------- |
+| `heading` | String    | Title of the Content Card item.                            |
+| `copy`    | String    | Copy of the Content Card item.                             |
+| `cta`     | Object    | Object containing target and href of cta. See `cta` below. |
+
+### cta
+
+| Name   | Data Type | Description                       |
+| ------ | --------- | --------------------------------- |
+| `href` | String    | Url of the Content Card item cta. |
+
+## Stable selectors
+
+| Name                           | Description |
+| ------------------------------ | ----------- |
+| `dds--contentgroupcards`       | Component   |
+| `dds--contentgroupcards-group` | Component   |
+| `dds--contentgroupcards-item`  | Component   |
```

Comment:
Along with class names, let's get a consistent naming convention for the content group and all of its decorator patterns for the stable selector names.

Reply:
I've updated the documentation and the components with the naming convention, thanks!